<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planRegister</class>
 <widget class="QDialog" name="planRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plan Register</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>16</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>42</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>71</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>120</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>120</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwPlayer">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>13</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>120</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAge">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>69</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40+</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbPosition">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>40</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Forward</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Midfielder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Defender</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>13</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>99</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
